sdk/core/src/errors.rs

```diff
@@ -16,9 +16,9 @@ pub enum ParsingError {
 #[non_exhaustive]
 #[derive(Debug, Clone, PartialEq, thiserror::Error)]
 pub enum ParseError {
-    #[error("Expected token \"{}\" not found", 0)]
```

Comment:
Any reason not to include `ElementNotFound` from `ParsingError` in here? Seems a little like splitting hairs for a distinction most people wouldn't make.